tests/pipeline/test_slice.py

```diff
@@ -221,7 +221,7 @@ def compute(self, today, assets, out, col):
         my_unsafe_factor_slice = my_unsafe_factor[my_asset]
 
         class UsesSlicedInput(CustomFactor):
-            window_length = 1
+            window_length = 2
```

Comment:
We should update `MySafeFactor` below as well so that it's still testing that we allow a potentially-unsafe factor to be run over a window-safe slice.

Reply:
@ssanderson fixed in most recent commit